plio/views.py

```diff
@@ -128,8 +125,15 @@ def list_uuid(self, request):
         if page is not None:
             return self.get_paginated_response(page)
 
+        # return an empty response in a paginated format if pagination fails
         return Response(
-            {"detail": "Cannot paginate plio uuids"}, status=status.HTTP_404_NOT_FOUND
+            {
+                "count": 0,
+                "page_size": self.get_page_size(self.request),
```

Comment:
shouldn't this be 0?

Reply:
Nhi page size has already been configured in the Paginator class. Writing it 0 here will be incorrect